<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>523</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>4000</width>
    <height>4000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="lblTitulo">
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btnUpdateCustomers">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Update Customers</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btnDeleteLine">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color: rgb(85, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Delete Line</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btnDeleteAllCustomers">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color: rgb(255, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Delete ALL Customers</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btnEditLine">
      <property name="text">
       <string>Edit Line</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="resourcers.qrc">
         <normaloff>:/imgs/imgs/add.png</normaloff>:/imgs/imgs/add.png</iconset>
       </attribute>
       <attribute name="title">
        <string>New</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Name">
        <property name="geometry">
         <rect>
          <x>61</x>
          <y>9</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>37</y>
          <width>46</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Surname:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SurNames">
        <property name="geometry">
         <rect>
          <x>61</x>
          <y>37</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>65</y>
          <width>34</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Phone">
        <property name="geometry">
         <rect>
          <x>61</x>
          <y>65</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAdd">
        <property name="geometry">
         <rect>
          <x>261</x>
          <y>100</y>
          <width>90</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnConfigDataBase">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>233</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>  Config</string>
        </property>
        <property name="icon">
         <iconset resource="resourcers.qrc">
          <normaloff>:/imgs/imgs/if_database_edit_84524.png</normaloff>:/imgs/imgs/if_database_edit_84524.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset resource="resourcers.qrc">
         <normaloff>:/imgs/imgs/customer.png</normaloff>:/imgs/imgs/customer.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Customers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="cursor" stdset="0">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset resource="resourcers.qrc">
         <normaloff>:/imgs/imgs/search.png</normaloff>:/imgs/imgs/search.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" colspan="3">
         <widget class="QTextBrowser" name="textBrowser_2">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_Search"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Customer"/>
    <addaction name="actionShow_All_Customers"/>
    <addaction name="actionDelete_All_Customers"/>
    <addaction name="actionSearch_a_Customer"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Customer">
   <property name="icon">
    <iconset resource="resourcers.qrc">
     <normaloff>:/imgs/imgs/add.png</normaloff>:/imgs/imgs/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Customer</string>
   </property>
  </action>
  <action name="actionShow_All_Customers">
   <property name="icon">
    <iconset resource="resourcers.qrc">
     <normaloff>:/imgs/imgs/customer.png</normaloff>:/imgs/imgs/customer.png</iconset>
   </property>
   <property name="text">
    <string>Show All Customers</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDelete_All_Customers">
   <property name="icon">
    <iconset resource="resourcers.qrc">
     <normaloff>:/imgs/imgs/delete.png</normaloff>:/imgs/imgs/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete All Customers</string>
   </property>
  </action>
  <action name="actionSearch_a_Customer">
   <property name="icon">
    <iconset resource="resourcers.qrc">
     <normaloff>:/imgs/imgs/search.png</normaloff>:/imgs/imgs/search.png</iconset>
   </property>
   <property name="text">
    <string>Search a Customer</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resourcers.qrc">
     <normaloff>:/imgs/imgs/exit.png</normaloff>:/imgs/imgs/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourcers.qrc"/>
 </resources>
 <connections/>
</ui>
